src/discussions/utils.js

```diff
@@ -228,13 +228,35 @@ export function postMessageToParent(type, payload = {}) {
 export const isPostPreviewAvailable = (htmlNode) => {
   const containsImage = htmlNode.match(/(<img((?:\\.|.)*)>)/);
   const isLatex = htmlNode.match(/(\${1,2})((?:\\.|.)*)/)
-                  || htmlNode.match(/(\[mathjax](.+?))+/)
-                  || htmlNode.match(/(\[mathjaxinline](.+?))+/)
-                  || htmlNode.match(/(\\\[(.+?))+/)
-                  || htmlNode.match(/(\\\((.+?))+/);
+    || htmlNode.match(/(\[mathjax](.+?))+/)
+    || htmlNode.match(/(\[mathjaxinline](.+?))+/)
+    || htmlNode.match(/(\\\[(.+?))+/)
+    || htmlNode.match(/(\\\((.+?))+/);
 
   if (containsImage || isLatex || htmlNode === '') {
     return false;
   }
   return true;
 };
+
+/**
+ * Helper function to make a check if date is in given range
+ * @param {Date} date this date to be checked in range
+ * @param {Date} start start date
+ * @param {Date} end end date
+ */
+export function dateInDateRange(date, start, end) {
+  return date > start && date < end;
```

Comment:
This condition will exclude the blackout start and end date.  e.g blackout date starts from `3-08-2022` and the current date also is `3-08-2022`. '3-08-2022' > '3-08-2022' will be false. However, It is on a blackout date.
So condition should be like 
`date >= start && date =< end`